tests/schemas/rules/value/map.json

```diff
@@ -26,6 +26,30 @@
       ]
     }
   },
+  {
+    "description": "Map over a list of dates using `date-range` operator using a dynamic date source `now`",
```

Comment:
Map is opposite to what it usually would be in Python. I might have been tempted to explicitly express Map rather than assume knowledge

Reply:
I don't follow. Can you elaborate?.

Reply:
I look at it again and it is the right way around